packages/azure/kibana/dashboard/azure-5ee36c30-32dc-11ed-a2e6-916b60bbea71.json

```
@@ -1,748 +1,809 @@
-                "id": "logs-*",
-                "name": "indexpattern-datasource-layer-2b0a18e6-25a6-40ef-ade8-5dddbd897856",
-                "type": "index-pattern"
-              }
-            ],
-            "state": {
-              "datasourceStates": {
-                "formBased": {
-                  "layers": {
-                    "2b0a18e6-25a6-40ef-ade8-5dddbd897856": {
-                      "columnOrder": [
-                        "a1c2991d-9586-4c81-893a-e29584070568",
-                        "bd75d029-b44e-4c22-b06e-df0e4da1e694"
-                      ],
-                      "columns": {
-                        "a1c2991d-9586-4c81-893a-e29584070568": {
-                          "customLabel": true,
-                          "dataType": "string",
-                          "isBucketed": true,
-                          "label": "Source",
-                          "operationType": "terms",
-                          "params": {
-                            "missingBucket": false,
-                            "orderBy": {
-                              "columnId": "bd75d029-b44e-4c22-b06e-df0e4da1e694",
-                              "type": "column"
+    "attributes": {
+        "description": "Provide an overview and statistics of the provisioning activities on your enterprise applications.",
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": {
+                "filter": [
+                    {
+                        "$state": {
+                            "store": "appState"
+                        },
+                        "meta": {
+                            "alias": null,
+                            "disabled": false,
+                            "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
+                            "key": "data_stream.dataset",
+                            "negate": false,
+                            "params": {
+                                "query": "azure.identityprotection"
                             },
-                            "orderDirection": "desc",
-                            "otherBucket": true,
-                            "size": 5
-                          },
-                          "scale": "ordinal",
-                          "sourceField": "azure.identityprotection.properties.source"
+                            "type": "phrase"
                         },
-                        "bd75d029-b44e-4c22-b06e-df0e4da1e694": {
-                          "dataType": "number",
-                          "isBucketed": false,
-                          "label": "Count of records",
-                          "operationType": "count",
-                          "scale": "ratio",
-                          "sourceField": "___records___"
+                        "query": {
+                            "match_phrase": {
+                                "data_stream.dataset": "azure.identityprotection"
```

Update dataset to fix the dashboard:

```suggestion
                                "data_stream.dataset": "azure.identityprotection"
```

---

Here also shouldn't it be `"data_stream.dataset": "azure.identity_protection"`?

---

Yep, definitely! I probably mis-pressed the underscore key! 🤦  Not it is fixed as `azure.identity_protection`.